tests/test_clients.py

```diff
@@ -140,13 +158,47 @@ def test_import_without_config_file(self, vo):
         thus have to disable the access to it (move it) and make sure to run the
         code in a different environment.
         """
-        configfile = Config().configfile
-        rename(configfile, f"{configfile}.tmp")
-        try:
-            exitcode, _, err = execute("python -c 'from rucio.client import Client'")
-            print(exitcode, err)
-            assert exitcode == 0
-            assert "RuntimeError: Could not load Rucio configuration file." not in err
-        finally:
-            # This is utterly important to not mess up the environment.
-            rename(f"{configfile}.tmp", configfile)
```

Comment:
This is a pretty ominous comment: `# This is utterly important to not mess up the environment.`

The code block was added here: https://github.com/rucio/rucio/commit/9f78b001a1468e32538fb2afbfb534b3ceb91451

I think this is not needed anymore in this context, though

Reply:
Yeah I thought so as well. But I was too amused by the severity of the comment to remove it haha.